_includes/footer.html

```diff
@@ -45,3 +45,14 @@ <h5>Our Social Ocean</h5>
     </div>
   </div>
 </footer>
+
+<script src="https://unpkg.com/mermaid@8.9.3/dist/mermaid.min.js"></script>
```

Comment:
I used this version b/c it is closer to the one on GitHub and easier to port the chart we create there.


```suggestion
<script src="https://unpkg.com/mermaid@10.7.0/dist/mermaid.min.js"></script>
```

Reply:
Also, we probably don't need this here. Only in the `mermaid.html`.

Reply:
Ah! I forgot, we do need this here (or in the `_includes/main_header.html`).

```
{% if page.mermaid %}
  {% include mermaid.html %}
{% endif %}
```

That way only pages that have the `mermaid: true` in the front matter will trigger the mermaid JS.

Reply:
I don't see `_includes/main_header.html` is that a new file?

Reply:
I edited the code sprint jekyll theme. The footer should be fine for this one. As long as this ends in the body of the HTML we are good.